### PR TITLE
[Fix][ADF-542]: Add data-max with the value of maximum

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-542

**Related PR's**

- https://github.com/oat-sa/tao-core-ui-fe/pull/327
- https://github.com/oat-sa/extension-tao-itemqti/pull/1822

### CODE REVIEW

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-clagosarias.playground.kitchen.it.taocloud.org:46571/

File: views/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "1.15.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.27.5",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#fix/ADF-542/human-response-processing-minmax",
```
Comment (404passionFruit) [inline views/package.json:21]: Please do not forget to update to a proper version here

**Review by 404passionFruit — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`
- [x] Pull request title and description are meaningful
- [x] Commits are following conventional commits
- [x] Kitchen links are not exposed in the description

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

**Pull request merged**